pwndbg/commands/config.py

```diff
@@ -142,8 +150,8 @@ def themefile(show_all=False) -> None:
     configfile_print_scope("theme", show_all)
 
 
-def configfile_print_scope(scope, show_all=False) -> None:
-    params = pwndbg.gdblib.config.get_params(scope)
+def configfile_print_scope(scope: str, show_all: bool = False) -> None:
+    params = pwndbg.gdblib.config.config.get_params(scope)
```

Comment:
```suggestion
    params = pwndbg.gdblib.config.get_params(scope)
```

This works for me:
```
In [3]: pwndbg.gdblib.config.get_params
Out[3]: <bound method Config.get_params of <pwndbg.lib.config.Config object at 0x7fa5289d6980>>
```

Reply:
yes, it works as they have been copied to the global scope of that file. but in reality

`pwndbg/gdblib/config.py`

in this file there is no `get_param` but it's here
`config = pwndbg.lib.config.Config()`
this config has get_param.

Anyways, I have fixed the suggested change.

Reply:
fixed!